lib/ChainwebDb/Types/Transfer.hs

```diff
@@ -0,0 +1,60 @@
+{-# LANGUAGE DeriveAnyClass #-}
+{-# LANGUAGE DeriveGeneric #-}
+{-# LANGUAGE DerivingStrategies #-}
+{-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE FlexibleInstances #-}
+{-# LANGUAGE GeneralizedNewtypeDeriving #-}
+{-# LANGUAGE ImpredicativeTypes #-}
+{-# LANGUAGE MultiParamTypeClasses #-}
+{-# LANGUAGE NoImplicitPrelude #-}
+{-# LANGUAGE TypeFamilies #-}
+{-# LANGUAGE UndecidableInstances #-}
+
+{-# OPTIONS_GHC -fno-warn-missing-signatures #-}
+
+module ChainwebDb.Types.Transfer where
+
+----------------------------------------------------------------------------
+import BasePrelude
+import Data.Text (Text)
+import Database.Beam
+------------------------------------------------------------------------------
+import ChainwebDb.Types.Block
+import ChainwebDb.Types.Common
+------------------------------------------------------------------------------
+data TransferT f = Transfer
+  { _tr_block :: PrimaryKey BlockT f
+  , _tr_requestkey :: C f ReqKeyOrCoinbase
+  , _tr_chainid :: C f Int64
+  , _tr_height :: C f Int64
+  , _tr_idx :: C f Int64
+  , _tr_modulename :: C f Text
+  , _tr_moduleHash :: C f Text
+  , _tr_from_acct :: C f Text
+  , _tr_to_acct :: C f Text
+  , _tr_amount :: C f Double
+  }
+  deriving stock (Generic)
+  deriving anyclass (Beamable)
+
+Transfer
+  (BlockId (LensFor tr_block))
+  (LensFor tr_requestkey)
+  (LensFor tr_chainid)
+  (LensFor tr_height)
+  (LensFor tr_idx)
+  (LensFor tr_modulename)
+  (LensFor tr_moduleHash)
+  (LensFor tr_from_acct)
+  (LensFor tr_to_acct)
+  (LensFor tr_amount)
+ = tableLenses
```

Comment:
It has been the standard for the other tables to include these lenses. I don't necessarily see the harm in keeping them around. But I'll let you decide what's best here since I'm not really leaning one way or the other.